modules/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md

```diff
@@ -138,6 +138,7 @@ tags: {
 | :-- | :-- | :-- |
 | `location` | string | The location the resource was deployed into. |
 | `name` | string | The name of the user assigned identity. |
+| `clientId` | string | The client ID of the user assigned identity. |
```

Comment:
@davidkarlsen please update the readme as per [guidance](https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Generate%20module%20Readme). And also please attach a successful module pipeline run as well to this PR